samples/nextjs/package.json

```diff
@@ -76,7 +76,6 @@
     "eslint-plugin-react": "^7.21.5",
     "eslint-plugin-yaml": "^0.2.0",
     "graphql-let": "^0.16.2",
-    "graphql-tag": "~2.10.1",
```

Comment:
@ambrauer Did you run nextjs sample after removing of `graphql-tag`? I remember that it's required by graphql-let.
https://github.com/piglovesyou/graphql-let/blob/main/package.json#L140

Reply:
@illiakovalenko This is actually a duplicate - we also have `graphql-tag` under `dependencies` (which was for a newer version and is still present).

Reply:
Got it